pykickstart/commands/timezone.py

```diff
@@ -35,6 +35,12 @@ def __init__(self, writePriority=0, *args, **kwargs):
         self.isUtc = kwargs.get("isUtc", False)
         self.timezone = kwargs.get("timezone", "")
 
+        # tracks if the timezone command requires a timezone specififcation
+        # - this has been the case up to the F18 version of the command
+        # - this variable has been added to make it possible for later versions
+        #   of the command to drop this requirement
+        self._timezone_spec_required = True
```

Comment:
This would make more sense to me as a class attribute. But in general, I don't think these changes are worth reusing 3 lines of code in the ``parse()`` method. The RHEL7 version of the command could just have its own ``parse()`` method defined instead.